### DISCUSSION TITLE
Earthquake Event Simulation: error for forecast rupture scenarios

### DISCUSSION THREAD

**Discussion by tingerzhu** [NHERI-SimCenter/SimCenterCommon#247] [R2D] [answered]
Dear R2D support team,

I am trying to run the ground failure model for the Bay Area based on a M7.2 Hayward Earthquake scenario (https://earthquake.usgs.gov/scenarios/eventpage/nclegacyhaywardrodgerscreekrchnm7p2_se/executive). I used the point source function to input the scenario information but ran into the attached error when forecasting the rupture scenario. May I know what I can do to resolve this error? Or is there an alternative way to run the ground failure model given the origin information and corresponding ShakeMap (PGA, PGV, PSA)? Thanks!

![error message](https://github.com/user-attachments/assets/d4c41374-965e-414e-96dd-d0530158332b)

![R2D input](https://github.com/user-attachments/assets/fa7a8a18-3eeb-4f99-8033-81cf9adf46da)

**Comment by jinyan1214** [comment, marked as answer]
Thank you very much for your interest in using R2D. 

The error you posted on using the rupture forecasting model is likely because your computer is not connected to the internet or the opensha server is not responding. Please make sure you are connected and let us know if the error persists. We will be happy to help if any further errors appear.

Using ShakeMap ground shaking intensity measures are not supported through the user interface now. You will need to try your own python interface to the SimCenter liquefaction and landslide backend code at [here](https://github.com/NHERI-SimCenter/SimCenterBackendApplications/blob/e8860ecb31cc6f8b13d6bfbfb9e2955d6fcdc2ea/modules/performRegionalEventSimulation/regionalGroundMotion/HazardSimulationEQ.py#L433) and [here](https://github.com/NHERI-SimCenter/SimCenterBackendApplications/blob/e8860ecb31cc6f8b13d6bfbfb9e2955d6fcdc2ea/modules/performRegionalEventSimulation/regionalGroundMotion/liquefaction.py#L1).

**Reply by tingerzhu** [reply]
It works now. Thank you!
